ui/tests/test_views.py

```diff
@@ -22,14 +23,57 @@
 }
 
 
+@pytest.fixture
+def search_results():
+    return [{'label': 'company', 'id': 1}]
+
+
+@pytest.fixture
+def request_search_valid_bbc(rf):
+    url = reverse('onboarding-find-company')
```

Comment:
on the other hand this is 'directory-form', meaning directory-register-form, so probably namespacing with it is not necessary at all, 'find-company' could be enough.

I think we might actually fork this and rename 'directory-ui' and directory-form-data will become 'directory-api' or 'directory-platform', following the 'start with a multi-tier monolith approach'.